.github/workflows/ci_gpu.yml

```diff
@@ -20,11 +20,10 @@ jobs:
       - name: Clean up previous run
         run: |
           echo "Cleaning up previous run"
-          cd $RUNNER_WORKSPACE
-          pwd
-          cd ..
-          pwd
-          rm -rf _tool
+          ls -la ./
+          sudo rm -rf ./* || true
+          sudo rm -rf ./.??* || true
+          ls -la ./
```

Comment:
we need specify the path and then delete

Reply:
This doesn't work https://github.com/pytorch/serve/actions/runs/6409297939

Also, we have been doing the same in Regression GPU runners and it works     

https://github.com/pytorch/serve/blob/master/.github/workflows/regression_tests_gpu.yml#L21

Reply:
Could you maybe add a comment to more explicitly say what this is trying to achieve?

The benefit of not having `*` in remove commands is we can be more explicit about what's being deleted